plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedTelemetryUtils.kt

```diff
@@ -70,6 +71,36 @@ fun getEnabledConnectionsForTelemetry(project: Project?): Set<AuthFormId> {
 fun getEnabledConnections(project: Project?): String =
     getEnabledConnectionsForTelemetry(project).joinToString(",")
 
+fun getAuthScopesForTelemetry(project: Project?): Set<String> {
```

Comment:
The function above repeats some of the logic used here, can we move the common logic to a function?